comfort-service.yml

```diff
@@ -0,0 +1,298 @@
+          - name: component
+            description: |
+              The component position to change
+            type: movement.seat_component_t
+          - name: position
+            description: |
+              The desired position to move the component to
+            type: movement.movement_t
+
+      - name: current_position
+        description: |
+          Get the current position of the seat
+
+        in_arguments:
+          - name: row
+            description: |
+              The desired seat to row query
+            type: uint8
+          - name: index
+            description: |
+              The desired seat index to query
+            type: uint8
+
+        out_arguments:
+          - name: seat
+            description: |
+              The seat state that was requested
+            type: movement.seat_t
+
+    # EVENTS
+    #
+    # This section defines all events for the given namespace.
+    #
+    # Events are non-returning commands with multiple subscribers,
+    # much like SOME/IP events.
+    # Unlike signals an event retains no state (value) after it has
+    # been called; once it has been sent it is forgotten.
+    #
+    # TODO:
+    # + FrancaIDL may lack semantics to cover this. We need to
+    #   create apropriate transformation rules.
+    #
+    events:
+      - name: seat_moving
+        description: |
+          The event of a seat beginning movement
+        in_arguments:
+          - name: status
+            description: |
+              The movement status, moving (1), not moving (0)
+            type: uint8
+          - name: row
+            description: |
+              The row of the seat
+            type: uint8
+          - name: index
+            description: |
+              The index of the seat position in the row
+            type: uint8
+          - name: component
+            description: |
+              The seat component that is moving
+            type: movement.seat_component_t
+
+      - name: passenger_present
+        description: |
+          When the seat passenger status changes
+        in_arguments:
+          - name: status
+            description: |
+              The status of seat passenger, passenger (1), no passenger (0)
+            type: uint8
+          - name: row
+            description: |
+              The row of the seat
+            type: uint8
+          - name: index
+            description: |
+              The index of the seat position in the row
+            type: uint8
+
+
+    # PROPERTIES
+    #
+    # This section defines properties for the given namespace.
+    #
+    # Properties are service data elements that are distributed
+    # across one or more subscribers. They are equivalent to
+    # MQTT topics or SOME/IP fields.
+    #
+    # TODO: Define relationship, rules, and semantics
+    #       between properties and VSS signals.
+    #       See vss_integration_proposal.yml for one solution.
+    #
+    properties:
```

Comment:
Franca calls them Attribute but I agree Property is a fairly well understood term.
Also, "attribute" has a different and conflicting meaning in VSS, so we should rename either one or the other for sure.

Reply:
The more general question is:  Is there a need for properties or should they be described in a "VSS-like model" and referred to instead?